### PR TITLE
Update description of Rex

### PR DESCRIPTION
**Pull request by ferki**
As discussed on various forums before, I propose updating the description of Rex to "friendly automation framework".

In this PR, I also included a short explanation of why we consider Rex to be friendly. If you prefer that, we can split the commit and just edit the heading for now.

I also did not include changes outside the README in this PR, as I feel like that would be best to handle separately. But again, if you prefer, we can try adding them here already.

There will be an accompanying PR for the website too.

@krimdomu @ehuelsmann: please review, and give feedback.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,27 @@
...
+
+    Automate what you are doing today, and add more tomorrow.
+
+    Rex runs locally, even if managing remotes via SSH. This means it's instantly usable, without big rollout processes or anyone else to convince, making it ideal and friendly for incremental automation.
+
+1. It's just perl
+
+    Perl is a battle tested, mature language, and Rex code is just perl code.
+
+    This means whenever you reach the limitations of the built-in Rex features, a powerful programming language and module ecosystem is always at your fingertips to seamlessly extend it with modules from CPAN or with your own code.
```
Comment (ehuelsmann) [inline README.md]: Should CPAN link to MetaCPAN ?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,27 @@
...
+    As a bonus, you can also use the usual well-established tools and workflows, like IDE integration for syntax highlighting, linting and formatting, or authoring and publishing Rex modules to CPAN.
```
Comment (ehuelsmann) [inline README.md]: Link "rex modules on cpan" to a metaCPAN query?

File: README.md (hunk 1 above)
Comment (ferki) [reply README.md]: Link added.

File: README.md (hunk 2 above)
Comment (ferki) [reply README.md]: Reworded `s/to/on/` and added link.

**Comment by ferki** [comment]
@ehuelsmann: thanks for the feedback, I added links :+1:

**Pull request merged**